tests/integration_tests/functional/test_cmd_line_start.py

```diff
@@ -432,7 +432,7 @@ def test_config_start_and_mmds_with_api(uvm_plain, vm_config_file):
     assert response.json() == {}
 
     # Populate MMDS with data.
-    response = test_microvm.api.mmds.put(**data_store)
```

Comment:
I wonder why our Python linters don't catch these things

Reply:
I honestly have no idea. My IDE caught it, so it should be possible